packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -0,0 +1,149 @@
+import Button, { BtnSizes } from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { Fragment } from 'react'
+import { useAsync } from 'react-async-hook'
+import { ActivityIndicator, ScrollView, StyleSheet, Text } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch } from 'react-redux'
+import { showError } from 'src/alert/actions'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { CELO_REWARDS_LINK } from 'src/brandingConfig'
+import { ContentType, fetchConsumerRewardsContent } from 'src/consumerIncentives/contentFetcher'
+import i18n from 'src/i18n'
+import { headerWithCloseButton } from 'src/navigator/Headers'
+import { navigate, navigateBack } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import useTypedSelector from 'src/redux/useSelector'
+import { fetchI18nContent } from 'src/utils/translationsFetcher'
+
+const useConsumerIncentivesContent = () => {
+  const contentResult = useAsync<ContentType>(fetchConsumerRewardsContent, [])
+  let texts
+  if (contentResult.result) {
+    texts = fetchI18nContent(contentResult.result)
+  }
+  return {
+    content: texts,
+    loading: contentResult.loading,
+    error: contentResult.loading,
+  }
+}
+
+const range = (n: number) => (n === 0 ? [] : [...Array(n).keys()].map((i) => i + 1))
+
+type Props = StackScreenProps<StackParamList, Screens.ConsumerIncentivesHomeScreen>
+export default function ConsumerIncentivesHomeScreen(props: Props) {
+  const userIsVerified = useTypedSelector((state) => state.app.numberVerified)
+  const { content, loading, error } = useConsumerIncentivesContent()
+  const dispatch = useDispatch()
+
+  if (!loading && error) {
+    dispatch(showError(ErrorMessages.FIREBASE_FETCH_FAILED))
+    navigateBack()
+    return null
+  }
```

Comment:
nit: seems a bit radical to navigate back 😄  I know we're short on time, but would suggest checking the behavior with Cal / Corey and follow up in another PR if needed.

Reply:
Sounds good. Hopefully it's not too frequent, but I did check with the team and it looks like we'll add a retry button of some kind in 1.10